Dockerfile.ppc64le

```diff
@@ -82,7 +82,21 @@ RUN cd /usr/local/lvm2 \
 # TODO install Go, using gccgo as GOROOT_BOOTSTRAP (Go 1.5+ supports ppc64le properly)
 # possibly a ppc64le/golang image?
 
-ENV PATH /go/bin:$PATH
+## BUILD GOLANG 1.6
+ENV GOLANG_VERSION 1.6rc2
```

Comment:
This should probably be `GO_VERSION`, to be consistent with the other Dockerfiles (and makes it easier to find all versions in use) https://github.com/docker/docker/blob/master/Dockerfile.armhf#L78